doc/en/user/source/extensions/printing/index.rst

```diff
@@ -14,7 +14,11 @@ Reference:
 Installation
 ------------
 
-* Visit the :website:`website download <download>` page, locate your release, and download:  :download_extension:`printing`
+* Visit the :website:`website download <download>` page, locate your release, and download: 
+
+   * |release| :download_extension:`printing`
+   * |version| :nightly_extension:`printing`
+
```

Comment:
The instructions are confusing on two levels:
* First it says to go to the webside and find the release
* Then it provides a direct link

Besides the confusion, the direct link is not going to work for most, as they are running their own random version of GeoServer, which is going to be mismatched with the extension pointed by the link. Also from experience, the latest stable release is the least deployed version, because most installations do resist upgrades.

Suggestion: tell them what to do instead:
* Find the geoserver version number
* Find the release page
* Download the plugin from there

If it's a community, let them look at the nightly builds for the relevant series intead.